openstudiocore/src/model/WaterUseEquipment.cpp

```diff
@@ -227,6 +227,11 @@ WaterUseEquipment::WaterUseEquipment(const WaterUseEquipmentDefinition& waterUse
   : SpaceLoadInstance(WaterUseEquipment::iddObjectType(),waterUseEquipmentDefinition)
 {
   OS_ASSERT(getImpl<detail::WaterUseEquipment_Impl>());
+
+  Schedule sch = this->model().alwaysOnDiscreteSchedule();
```

Comment:
@jmarrec why are you doing this here?  seems like this is more than we need to do?  wouldn't a blank here be covered by a default schedule?

Reply:
i see, this was for inspector and this is the default value